### PR TITLE
Problem: integration test doc don't mention contract compilation

### PR DESCRIPTION
**Pull request by yihuang**
Solution:
- update integration test doc

👮🏻👮🏻👮🏻 !!!! REFERENCE THE PROBLEM YOUR ARE SOLVING IN THE PR TITLE AND DESCRIBE YOUR SOLUTION HERE !!!! DO NOT FORGET !!!! 👮🏻👮🏻👮🏻

# PR Checklist:

- [ ] Have you read the [CONTRIBUTING.md](https://github.com/crypto-org-chain/chain-main/blob/master/CONTRIBUTING.md)?
- [ ] Does your PR follow the [C4 patch requirements](https://rfc.zeromq.org/spec:42/C4/#23-patch-requirements)?
- [ ] Have you rebased your work on top of the latest master? 
- [ ] Have you checked your code compiles? (`make`)
- [ ] Have you included tests for any non-trivial functionality?
- [ ] Have you checked your code passes the unit tests? (`make test`)
- [ ] Have you checked your code formatting is correct? (`go fmt`)
- [ ] Have you checked your basic code style is fine? (`golangci-lint run`)
- [ ] If you added any dependencies, have you checked they do not contain any known vulnerabilities? (`go list -json -m all | nancy sleuth`)
- [ ] If your changes affect the client infrastructure, have you run the integration test?
- [ ] If your changes affect public APIs, does your PR follow the [C4 evolution of public contracts](https://rfc.zeromq.org/spec:42/C4/#26-evolution-of-public-contracts)?
- [ ] If your code changes public APIs, have you incremented the crate version numbers and documented your changes in the [CHANGELOG.md](https://github.com/crypto-org-chain/chain-main/blob/master/CHANGELOG.md)?
- [ ] If you are contributing for the first time, please read the agreement in [CONTRIBUTING.md](https://github.com/crypto-org-chain/chain-main/blob/master/CONTRIBUTING.md) now and add a comment to this pull request stating that your PR is in accordance with the [Developer's Certificate of Origin](https://github.com/crypto-org-chain/chain-main/blob/master/CONTRIBUTING.md#developer-certificate-of-originn).

Thank you for your code, it's appreciated! :)

### CODE REVIEW

File: docs/integration-test.md [hunk 1]
```diff
@@ -38,20 +45,47 @@ make run-integration-tests
 
 ## Customize The Test Runner
 
+To run customize test runner, you can also issue commands separatedly.
```
Comment (yihuang) [inline docs/integration-test.md:48]:
```suggestion
To customize test runner, you can also issue commands separatedly.
```

File: docs/integration-test.md [hunk 2]
```diff
@@ -38,20 +45,47 @@ make run-integration-tests
...
+### Run `pytest`
+
 We use `pytest` to discover the test cases and run them, follow [pytest doc](https://docs.pytest.org/en/6.2.x/contents.html) for more options.
 
 You can invoke `pytest` after entering the nix shell:
 
 ```shell
-$ nix-shell integration_tests/shell.nix
-<nix-shell> $ pytest
+$ pytest -s -vv ./integration_tests
```
Comment (yihuang) [inline docs/integration-test.md:75]:
```suggestion
$ cd integration_tests
$ pytest -s -vv
```

File: docs/integration-test.md [hunk 3]
```diff
@@ -38,20 +45,47 @@ make run-integration-tests
...
 ```
 
 You can use `-k` to select test cases by patterns in name:
 
 ```shell
-<nix-shell> $ # run against cronos only
-<nix-shell> $ pytest -k cronos
-<nix-shell> $ # run against geth only
-<nix-shell> $ pytest -k geth
+$ pytest ./integration_tests -k test_basic
```
Comment (yihuang) [inline docs/integration-test.md:81]:
```suggestion
$ cd integration_tests
$ pytest -k test_basic
```

File: docs/integration-test.md [hunk 4]
```diff
@@ -38,20 +45,47 @@ make run-integration-tests
...
+```
+
+Some test cases will run on both `geth` and `cronos`, you can also select the platform to run using `-k`:
+
+```shell
+$ # run against cronos only
+$ pytest ./integration_tests -k cronos
+$ # run against geth only
+$ pytest ./integration_tests -k geth
```
Comment (yihuang) [inline docs/integration-test.md:90]:
```suggestion
$ cd integration_tests
$ # run against cronos only
$ pytest -k cronos
$ # run against geth only
$ pytest -k geth
```

**Review by thomas-nguy — APPROVED** [review]
lgtm

I had few issues though that I manage to solve by changing manually owners of contracts file (Error: EACCES: permission denied) but its maybe a local issue

**Pull request merged**